src/main/res/layout/oath_code_view.xml

```diff
@@ -30,27 +30,53 @@
   ~ SUCH DAMAGE.
   -->
 
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                android:orientation="horizontal"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:padding="12dp"
-                style="@style/activated"
-    >
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="horizontal"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="12dp"
+    android:baselineAligned="false"
+    style="@style/activated">
+
+    <LinearLayout
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:layout_weight="1">
+
+        <TextView
+            android:layout_width="fill_parent"
+            android:layout_height="wrap_content"
+            android:text="Issuer"
+            android:visibility="visible"
```

Comment:
Shouldn't this be "gone" or "invisible"? "Invisible" has the downside that it shows empty space, but "gone" means different items have different heights. Out of the two I think I prefer "gone", as it's less noticeable. I'll merge once this is amended!